src/sql/platform/connection/common/providerConnectionInfo.ts

```diff
@@ -197,13 +197,16 @@ export class ProviderConnectionInfo extends Disposable implements azdata.Connect
 			return false;
 		}
 
+		if (this.providerName === 'KUSTO') {
```

Comment:
Why is kusto special for this anyways? The kusto provider should be returning the correct value for isRequired I would think